### DISCUSSION TITLE
Thank you for mentioning "Object Thinking"!

### DISCUSSION THREAD

**Discussion by Phundamentals** [MarimerLLC/csla#3515] [General]
I started reading the introductory "Using CSLA 2019" e-book. You mentioned another book, "Object Thinking" by David West. I bought that book and read about one third of it, now. Thank you so much for mentioning it! This will most likely help me overcome the difficulties I've encountered with software design and enable me to progress again.

Just needed to say that.

**Comment by rockfordlhotka** [comment]
You are welcome. I really found "Object Thinking" to be valuable when I was learning (re-learning?) how to properly do object-oriented design and programming.

**Reply by Phundamentals** [reply]
Definitely **re**-learning on my side!
